data/presets/presets/craft/metal_construction.json

```diff
@@ -7,5 +7,5 @@
     "tags": {
         "craft": "metal_construction"
     },
-    "name": "Metal Construction"
+    "name": "Metalworker"
```

Comment:
You should be able to merge the latest `develop` into your branch to resolve these unrelated changes.